docs/operational_guide/placement.md

```diff
@@ -0,0 +1,205 @@
+# Placement
+
+## Overview
+
+**Note**: The words *placement* and *topology* are used interchangeably throughout the M3DB documentation and codebase.
+
+A M3DB cluster has exactly one Placement. That placement maps the cluster's shard replicas to hosts. A cluster also has 0 or more namespaces, and each host serves every namespace for the shards it owns. In other words, if the cluster topology states that host A owns shards 1, 2, and 3 then host A will own shards 1, 2, 3 for all configured namespaces in the cluster.
+
+M3DB stores its placement (mapping of which hosts are responsible for which shards) in [etcd](https://coreos.com/etcd/). There are three possible states that each host/shard pair can be in:
+
+1. `Initializing`
+2. `Available`
+3. `Leaving`
+
+Note that these states are not a reflection of the current status of an M3DB node, but an indication of whether a given node has ever successfully bootstrapped and taken ownership of a given shard (achieved goal state). For example, in a new cluster all the nodes will begin with all of their shards in the `Initializing` state. Once all the nodes finish bootstrapping, they will mark all of their shards as `Available`. If all the M3DB nodes are stopped at the same time, the cluster placement will still show all of the shards for all of the hosts as `Available`.
+
+## Initializing State
```

Comment:
take a look at the rendering of this readme, you're missing a few block quotes in places -- https://github.com/m3db/m3/blob/a0b6410e8151927b128c8e68c3b5b1ac1bd0d15a/docs/operational_guide/placement.md

Reply:
fixed